yami-ichi.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: page
+order: 1
+title: "2018"
+subtitle: "Kickoff Yami-ichi"
+titleDisplay: "Yami-Ichi"
+excerpt:
+startDate: 2018-07-13
+redirect_from:
+  - '/2018/kickoff-yami-ichi/'
+---
+
+Friday • 13
+
+```
```

Comment:
Maybe this could be block quote instead with reference?

Reply:
Sure! Tho I think the amt of times July is on the homepage is a bit distracting -- maybe we swap?

Reply:
changed